doc/guide/cli_quickstart.hpp

```diff
@@ -200,21 +200,21 @@ easily plug into a data science production workflow for the command line.  A
 great thing to do next would be to look at more documentation for the mlpack
 command-line programs:
 
- - <a href="http://www.mlpack.org/docs/mlpack-git/man.html">mlpack
+ - <a href="http://www.mlpack.org/doc/mlpack-git/cli_documentation.html">mlpack
```

Comment:
I don't think the links in this file work.

Reply:
Ah, thanks for pointing this out.  Actually the issue was that the Doxygen nightly documentation had not properly fully rebuilt.  I updated my scripts and I think that it should not be a problem anymore.